src/Components/Banner/Banner.jsx

```diff
@@ -0,0 +1,24 @@
+import './Banner.css';
+
+function Banner(props) {
```

Comment:
You can destructure props object.
const {dispatch} = props;
And then use :
<img src={dispatch.image} className="" alt="" />
You have to read about destructuring here: https://learn.javascript.ru/destructuring-assignment#destrukturizatsiya-obekta